Extension/ui/settings.ts

```diff
@@ -260,6 +261,7 @@ class SettingsApp {
 
             (<HTMLInputElement>document.getElementById(elementId.intelliSenseMode)).value = config.intelliSenseMode ? config.intelliSenseMode : "${default}";
             (<HTMLInputElement>document.getElementById(elementId.includePath)).value = joinEntries(config.includePath);
+            (<HTMLInputElement>document.getElementById(elementId.dotConfig)).value = joinEntries(config.dotConfig);

```

Comment:
Can you move this to the end of the settings (to the Advanced section)? Same for the case above with the elementId.

Reply:
@sean-mcmanus proposed changes applied, thanks for the review.